main.py

```diff
@@ -0,0 +1,24 @@
+def menu():
+    """This function displays the menu and returns whether the user chose to play or not.
+
+    :return: True if the user chose to play, False otherwise."""
+    while True:
+        print("MENU \n1. New Game \n2. Exit")
+        choice = int(input())
```

Comment:
this raises an exception and crashes your program if the user enters a bad value. do proper error handling.